gammapy/utils/units.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Units and Quantity related helper functions"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import astropy.units as u
+
+__all__ = [
+    'standardise_unit',
```

Comment:
@cdeil - I think it depends a little on exactly how you want to apply it. If you know what output unit you want, I think @adrn's suggestion is best, as it is nice and explicit. If you simply want to remove all occurrences of `ph` and `ct`, then I think your approach is better.